README.md

```diff
@@ -257,14 +257,16 @@ directly deposited to your token balance.
 
 #### Minimize load on your node
 
-To start `flip` auctions, the keeper needs a list of urns and the collateralization ratio of each urn.  This list is 
-built by initializing an in-memory cache with urn history, and then querying recent events upon each block.  There are 
-two ways to initialize the cache:
- * **Set `--from-block` to the block where the first urn was created** to scrape the chain for `frob` events. 
+To start `flip` auctions, the keeper needs a list of urns and the collateralization ratio of each urn.  There are 
+two ways to retrieve the list of urns:
+ * **Set `--from-block` to the block where the first urn was created** to scrape the chain for `frob` events.  
```

Comment:
Consider adding a suggested block (`8928152`) in this description to relieve the friction of needing to find one as a reader. Using the `Vat` deployment block may not that intuitive.

Reply:
That's only a good number for ETH-A/BAT-A and mainnet.  If markdown supported tables, I'd be quick to add a matrix to the document.  I'm considering adding an image, as done elsewhere in the README.

Reply:
Done.